### ISSUE TITLE
Accessibility - Backoffice - Search input is missing label

### ISSUE THREAD

**Issue by rickdoesburg** [umbraco/Umbraco-CMS#21938]
### Which Umbraco version are you using?

17.2

### Bug summary

While doing a simplified backoffice WCAG audit I noticed a couple of WCAG issues.

When searching in the backoffice the input is missing a programmatically connected text label.

<img width="741" height="323" alt="Image" src="https://github.com/user-attachments/assets/77947707-a542-4707-a342-34ac81cec9ac" />

### Steps to reproduce

1. Press the search loop in the top bar to open search

### Expected result / actual result

To pass WCAG 3.3.2 Labels or Instructions the input field should have a visible label or hidden label with the same value as the placeholder (WCAG 2.5.3)

**Comment by andreaslborg** [comment]
Hi @rickdoesburg and @leekelleher,
I have made a PR for this issue: #21962. I hope this is the correct fix for the described issue.

**Comment by rickdoesburg** [comment]
@andreaslborg Awesome. This looks perfect.

**Comment by leekelleher** [comment]
@rickdoesburg @andreaslborg Thanks both for your contributions! 🎉 

Fixed in #21962 for v17.3.0.
